packages/eas-cli/src/metadata/utils/telemetry.ts

```diff
@@ -59,11 +51,13 @@ export function subscribeTelemetry(
     }
   );
 
-  function unsubscribeTelemetry(): void {
+  try {
+    return await action();
+  } catch (error: any) {
```

Comment:
In this case, it could throw any of these types: `MetadataUploadError | MetadataDownloadError | AssertError | TypeError`. I can cast it to `Error`, but that's not fully reflecting the types we could expect here.

(same goes with the other catch statements)